src/components/AccountDropdownComponent.jsx

```diff
@@ -312,10 +315,23 @@ class AccountDropdownComponent extends Component<Props, State> {
               })}
             {groupedCryptoAccounts.length !== 0 && <Divider />}
             <MenuItem value='addCryptoAccounts'>
-              <Button onClick={() => addAccount()} color='primary' fullWidth>
-                <AddIcon fontSize='small' />
+              {purpose === 'send' ? (
+                <Box display='flex' flex={1} justifyContent='space-between' alignItems='center' paddingTop='5px'>
```

Comment:
padding problem between elements on mobile (width = 411), should become stacked for mobiles according to @guyuxiang